packages/utils-worker/scripts/bundle-size.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
```

Comment:
Aren't we duplicating this script in a few places? Do we have an issue to refactor and just have it in one place?

Reply:
Yep. We didn't have an issue for this, but now we do: https://github.com/webhintio/hint/issues/3652